react/src/components/Map/Map.tsx

```diff
@@ -1,7 +1,86 @@
 import React from 'react';
+import { MapContainer, TileLayer, Marker, Popup } from 'react-leaflet';
+import MarkerClusterGroup from '@changey/react-leaflet-markercluster';
+import { TileServerLayer, FeatureCollection } from '../../types';
+import * as L from 'leaflet';
+import { LatLngTuple, MarkerCluster } from 'leaflet';
+import 'leaflet.markercluster';
 
-const Map = () => {
-  return <h2>Map</h2>;
+import 'leaflet/dist/leaflet.css';
+
+/* TODO: review if best approach is to style map with .leaflet-container */
```

Comment:
2 of these todos aren't linked with a jira issue.  i would usually prefer having a jira issue (so there are more notes and a reminder to do it) but maybe it is okay to note some things as todo and not have a jira issue as